quickref/quickref-ledger.md

```diff
@@ -9,7 +9,7 @@
 
 Last tested: 2016.
 See also:
-https://www.ledger-cli.org/3.0/doc/ledger3.html#Journal-Format,
+https://www.ledger-cli.org/doc/ledger3.html#Journal-Format,
```

Comment:
ℹ️ Since ledger 2.x is deprecated documentation for the (one and only ;) current version of ledger has moved to `doc/`. There is a redirect for `3.0/doc/` to `doc/`, yet it seems a good idea to update the url on this website.